Gemfile.template

```diff
@@ -6,10 +6,13 @@ gem "logstash-core", :path => "./logstash-core"
 gem "logstash-core-plugin-api", :path => "./logstash-core-plugin-api"
 
 gem "paquet", "~> 0.2"
-gem "pleaserun", "~>0.0.28"
-gem "rake", "~> 12"
-gem "ruby-progressbar", "~> 1"
+gem "pleaserun", "~>0.0.28", require: false
+gem "rake", "~> 12", require: false
+gem "ruby-progressbar", "~> 1", require: false
 gem "logstash-output-elasticsearch", ">= 11.6.0"
+gem "polyglot", require: false
+gem "treetop", require: false
+gem "faraday", "~> 1", :require => false # due elasticsearch-transport (elastic-transport) depending faraday '~> 1'
```

Comment:
I think these changes should come with [this PR](https://github.com/elastic/logstash/pull/14175)

Reply:
The only lines I'm seeing in this diff are my own. I rebased on top of `main`, and he lines you link were previously merged to `main` and are not changed by my PR.